src/ScrumBoardItBundle/Controller/DefaultController.php

```diff
@@ -18,25 +19,51 @@ class DefaultController extends Controller
 {
     /**
      * @Route("/", name="index")
-     * 
+     *
      * @return Response
      */
     public function indexAction()
     {
-        if ($this->get('security.authorization_checker')->isGranted('ROLE_AUTHENTICATED')) {
+        if ($this->get('security.authorization_checker')->isGranted('IS_CONFIGURED')) {
```

Comment:
Either rename the role of write a comment here as i don't understand this condition